netcon/includes/con_dll.h

```diff
@@ -768,7 +777,7 @@ int DLLGame_mode;
 char *DLLTracker_id;
 int *DLLNum_directplay_games;
 netgame_info *DLLNetgame;
-char *DLLLocalD3Dir;
+std::vector<std::filesystem::path> *DLLBase_directories;
```

Comment:
This variable should not be exported and accessible directly to netcon modules.

Reply:
OK. I removed that variable.